### PR TITLE
Suggestion: Needed a link while reading

### PR DESCRIPTION
**Pull request by ClydeDz**
For users reading this for the first time, would be great to have more context on what the 'Extensions and Updates' dialog is - adding a bit more context at the beginning of the article.

### CODE REVIEW

**Comment by PRMerger6** [comment]
@ClydeDz : Thanks for your contribution! The author, @gregvanl, has been notified to review your proposed change.

File: docs/extensibility/walkthrough-publishing-a-visual-studio-extension-via-command-line.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ ms.workload:
 ---
 # Walkthrough: Publishing a Visual Studio extension via command line
 
-This walkthrough shows you how to publish a Visual Studio extension to the Visual Studio Marketplace using the command line. When you add your extension to the Marketplace, developers can use the **Extensions and Updates** dialog to browse there for new and updated extensions.
+This walkthrough shows you how to publish a Visual Studio extension to the Visual Studio Marketplace using the command line. When you add your extension to the Marketplace, developers can use the [**Extensions and Updates**](https://docs.microsoft.com/visualstudio/ide/finding-and-using-visual-studio-extensions?view=vs-2017) dialog to browse there for new and updated extensions.
```
Comment (gregvanl) [inline docs/extensibility/walkthrough-publishing-a-visual-studio-extension-via-command-line.md]: For document relative links please use the following format:
[**Extensions and Updates**](../ide/finding-and-using-visual-studio-extensions.md).
You can find examples of this relative link format within the repo.

**Comment by PRMerger6** [comment]
@ClydeDz : Thanks for your contribution! The author, @gregvanl, has been notified to review your proposed change.

File: docs/extensibility/walkthrough-publishing-a-visual-studio-extension-via-command-line.md (hunk 1 above)
Comment (ClydeDz) [reply docs/extensibility/walkthrough-publishing-a-visual-studio-extension-via-command-line.md]: Thanks @gregvanl, I've added a new commit with the change.

**Comment by gregvanl** [comment]
#sign-off

**Pull request merged**